docs/src/Guides/03 Creating Commands.md

```diff
@@ -235,9 +237,9 @@ async def my_command_function(ctx: SlashContext, integer_option: int):
 
 For more information, please visit the API reference [here](/interactions.py/API Reference/API Reference/models/Internal/application_commands/#interactions.models.internal.application_commands.SlashCommandChoice).
 
-## I Need More Than 25 Choices
+## Autocompleting Choices
```

Comment:
This feels like a regression. 

Unless people already intimately know how discord works, there's an assumption that choices and autocomplete are the same thing. (UX-wise, to the end-user, they are)

"More than 25" is the most important part of this heading.  Feel free to rephrase it, but changing it to "Autocompleting choices" just makes people confused as to what choices without autocompletion does.

Reply:
I understand your point, however I'm not sure I agree with it. I think that the most important part of autocomplete choices is that they are dynamic, technically you're still limited to providing 25 max choices, so I'm not sure if "advertising" as allowing more than 25 choices is the best idea. Maybe others can join on this debate?

Reply:
I agree with Sil, this is a regression. At the very least an alternate like the following should be considered

> Autocomplete - when you need over 25 chouces